elasticdl/python/common/args.py

```diff
@@ -1,3 +1,44 @@
+from elasticdl.python.common.log_util import default_logger as logger
+
+MODEL_SPEC_GROUP = [
```

Comment:
@chunyang-wen @terrytangyuan is there any way to auto-fetch the list of parameters in Python? Little worried when we add new parameters but forget add here, it will cause inconsistency.

Reply:
Not on top of my head. Yes I am a little worried about this as well since this will easily get outdated if we forget to add them to the groups.

Reply:
All the arguments will get printed. The groups only help to print related arguments together.